<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>:)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  #B7CEE8;</string>
  </property>
  <widget class="QLabel" name="labelServerName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Адрес сервера БД</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatabaseName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Название БД</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelLogin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Имя Входа</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 15px;</string>
   </property>
   <property name="text">
    <string>Подключиться</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditServerName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 15px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Адрес сервера БД</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLogin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 15px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Имя Входа</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; border-radius: 15px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDatabaseName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>220</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	text-align: center; 
    border-radius: 15px;
    min-width: 6em;
	background: white;
}

QComboBox:editable {
    background: white;
}


QComboBox:on { /* shift the text when the popup opens */
    padding-top: 50px;
    padding-left: 50px;
	    background: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}


QComboBox QAbstractItemView {
    background: white; 
	border-radius: 10px;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
